src/main/java/frc/utils/CMDHandler.java

```diff
@@ -65,16 +65,17 @@ public static void runJavaClass(Class<?> classToRun, String... arguments) {
 		runJavaClass(pathOfClass, getSeparatedArguments(arguments));
 	}
 
-	public static void runPythonClass(Path pythonPath) {
-		runPythonClass(pythonPath, "");
+	public static void runPythonScript(Path pythonPath) {
+		runPythonScript(pythonPath, "");
 	}
 
 	/**
 	 * @param pythonPath The path from the java package to the class. example: "directory/example_class".
 	 * @param arguments  The arguments given to the python file.
 	 */
-	public static void runPythonClass(Path pythonPath, String... arguments) {
-		String command = "py " + pythonPath + ".py " + getSeparatedArguments(arguments);
+	public static void runPythonScript(Path pythonPath, String... arguments) {
+		String pythonName = isWindows() ? "py " : "python ";
+		String command = pythonName + pythonPath + ".py " + getSeparatedArguments(arguments);
```

Comment:
Script path should contain file suffix.
```suggestion
		String command = pythonName + pythonPath + " " + getSeparatedArguments(arguments);
```